mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/presenter/DebitCardPresenter.java

```diff
@@ -30,9 +32,22 @@ public void attachView(BaseView baseView) {
     @Override
     public void verifyDebitCard(String s, String s1, String s2) {
         String otp = "0000";
+        int expiryMonth = Integer.parseInt(s1);
+        int expiryYear = Integer.parseInt(s2);
+        int currentYear = Calendar.getInstance().get(Calendar.YEAR);
+        int currentMonth = Calendar.getInstance().get(Calendar.MONTH);
         if ((s.length() < 12) || (s.length() > 19)) {
             mDebitCardView.verifyDebitCardError(getContext()
-                    .getString(R.string.debit_card_error_message));
+                    .getString(R.string.debit_card_error_message), 1);
+        } else if (expiryMonth < 01 || expiryMonth > 12) {
+            mDebitCardView.verifyDebitCardError(getContext()
+                    .getString(R.string.expiry_month_error_message), 2);
+        } else if (expiryMonth <= currentMonth && expiryYear == currentYear) {
+            mDebitCardView.verifyDebitCardError(getContext()
```

Comment:
Add a condition if expiryMonth<currentMonth and expiryYear<=currentYear

Reply:
Ok, I will surely do that. Should I generate a new pull request after making the required changes? Also I don't know why this pull request failed the static analysis check although on my system it doesn't show any problem.

Reply:
@haran2248 I have made the changes you asked for. Can you please review this?